README.md

```diff
@@ -13,6 +13,17 @@ If you want to contribute projects or tools, please first read our [Contributing
 
 Submit a pull request to add a link to your repository here.
 
+### Tungsten demo day
+
+- [Bonus](https://github.com/seniorjoinu/Bonus) - Next-generation loyalty system built on the Internet Computer.
+
+- [Get Impact Now](https://github.com/getimpactnow/getimpactnow/) - Digital platform which enables responsible citizens to be a part of the decision-making process.
```

Comment:
```suggestion
- [Get Impact Now](https://github.com/getimpactnow/getimpactnow/) - Digital platform that enables responsible citizens to be a part of the decision-making process.
```
Not sure if that's quite an accurate description of the app. "responsible" is sort of value judgement and I think it is more about "interested" citizens and the app enables them to see and interact on issues that are important to their communities (where in theory they have the most impact and in in the world--I'm not sure how to capture that idea of the concentric circles of influence/impact)

Reply:
For these, I just added the descriptions that were from the teams' GitHub repos themselves.